cmd/dptp-controller-manager/main.go

```diff
@@ -43,7 +38,7 @@ func newOpts() (*options, error) {
 	flag.StringVar(&opts.configPath, "config-path", "", "Path to the prow config")
 	flag.StringVar(&opts.jobConfigPath, "job-config-path", "", "Path to the job config")
 	flag.StringVar(&opts.registryClusterKubeconfigPath, "registry-cluster-kubeconfig", "", "If set, this kubeconfig will be used to access registry-related resources like Images and ImageStreams. Defaults to the global kubeconfig")
-	flag.Var(&opts.promotionreconcilerOptions.IgnoredGitHubOrganizations, "promotionreconcilerOptions.ignored-github-organization", "GitHub organization to ignore in the imagestreamtagreconciler. Can be specified multiple times")
+	_ = flag.String("promotionreconcilerOptions.ignored-github-organization", "", "deprecated, doesn't do anything. Was used to ignore github organization. We instead attempt all repos and swallow 404 errors we get from github doing so.")
```

Comment:
Let's just remove the whole thing.

Reply:
That will send our deployment into crashlooping: https://github.com/openshift/release/blob/9e4b6fbfe9bc969a66e0b8b5a5a9844900021b44/clusters/app.ci/assets/dptp-controller-manager.yaml#L145

Reply:
Can you open a follow-up PR in release repo to remove that as well?

Reply:
Yup, that as the plan